app/qml/form/editors/MMFormGalleryEditor.qml

```diff
@@ -92,6 +92,7 @@ MMPrivateComponents.MMBaseInput {
         MouseArea {
           anchors.fill: parent
           onClicked: {
+            root.forceActiveFocus() //prevent app freeze as mentioned in issue #3538
```

Comment:
Isn't this happening also for the other relations editor?

Reply:
Additional question, is the crash happening only when taking a photo from within the child feature or is it reproducible for simple forms with an image editor?

Reply:
Indeed, the issue was also occurring in the `MMFormRelationReferenceEditor`. In the [bug report](https://github.com/MerginMaps/mobile/issues/3538) I also included the steps to reproduce the issue and added the solution in this pull request. 
I couldn't reproduce the crash when taking photos in simple forms with an image editor.